src/features/import-map.js

```diff
@@ -45,7 +45,15 @@ systemJSPrototype.resolve = function (id, parentUrl) {
 };
 
 function throwUnresolved (id, parentUrl) {
-  throw Error("Unable to resolve specifier '" + id + (parentUrl ? "' from " + parentUrl : "'"));
+  throw Error(errMsg(2, DEV ? "Unable to resolve specifier '" + id + (parentUrl ? "' from " + parentUrl : "'") : [id, parentUrl]));
+}
+
+function parseJson(script) {
+  try {
+    return JSON.parse(script);
+  } catch (err) {
+    throw Error(errMsg(1, DEV && "systemjs-import map contains invalid JSON"));
```

Comment:
I think it will minify better doing:

```js
if (DEV)
  throw Error(errMsg(1));
else
  throw Error(errMsg(1, 'SystemJS import map contains invalid JSON'));
```

but we should verify that too.